File: python/strings/SwapCase.go
// https://www.hackerrank.com/challenges/swap-case/problem

package main

import (
	"bufio"
	"fmt"
	"os"
	"unicode"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	text, _ := reader.ReadString('\n')
	a := []rune(text)
	for i, c := range a {
		if unicode.IsUpper(rune(c)) {
			a[i] = unicode.ToLower(c)
		} else {
			a[i] = unicode.ToUpper(c)
		}
	}
	res := string(a)
	fmt.Print(res)
}

File: python/strings/swap_case.py
# https://www.hackerrank.com/challenges/swap-case/problem

def swap_letter(c):
    return c.lower() if c.isupper() else c.upper()

def swap_case(s):
    s = list(s)
    l = list(map(swap_letter, s))
    return "".join(l)

if __name__ == '__main__':
    s = input()
    result = swap_case(s)
    print(result)
